docs/_advanced/custom_routes.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Adding Custom HTTP routes 
+lang: en
+slug: custom-routes
+order: 9
+---
+
+<div class="section-content">
+
+Adding custom HTTP routes is quite straightforward when using Bolt's built-in `ExpressReceiver`. Since `Bolt@v2.1.0`, `ExpressReceiver` added a `router` property, which exposes the Express [Router](http://expressjs.com/en/4x/api.html#router) on which more routes can be added.
+
+</div>
+
+```javascript
+const { App, ExpressReceiver } = require('@slack/bolt');
+
+// Create a Bolt Receiver
+const receiver = new ExpressReceiver({ signingSecret: '' });
```

Comment:
should we show `signingSecret` (and `token`) being read from `process.env.SLACK_SIGNING_SECRET`? i think this caused a little confusion in #514.

Reply:
Cleaned it up a bit to hopefully prevent that confusion.